### PR TITLE
Feat: Quizz

### PR DESCRIPTION
**Pull request by pritam-kr**
- map through all quiz category and display it
- context for quiz,
-  rules page for quiz
- fetched all quizzes

### CODE REVIEW

File: src/context/quizContext.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    payload: {
+        categories: DocumentData 
+    }
+}
+
+type GetQuizAction = {
+    type: "GET_QUIZ";
+    payload: {
+        quizzes: DocumentData 
+    }
```
Comment (msdeshmukh009) [inline src/context/quizContext.tsx:34]: You can extract this types

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: src/component/quizCard/QuizCard.tsx [hunk 2]
```diff
@@ -1,21 +1,36 @@
 
+import { useNavigate } from "react-router-dom"
+import { CategoryTypes } from "../dashboard/Dashboard"
+import "./quizCard.css"
+
+type props = {
+    category: CategoryTypes
```
Comment (varnita619) [inline src/component/quizCard/QuizCard.tsx:7]: props used nicely

**Review by varnita619 — APPROVED** [review]
Code is modular and clean
Ready to merge

**Pull request merged**